### ISSUE TITLE
[Bug] Deck.gl tries to override exception message which breaks in some exceptions

### ISSUE THREAD

**Issue by ricardoekm** [visgl/deck.gl#7945]
### Description

When a resource fetch is aborted a DOMException is thrown. Deck.gl tries to override the message property. However, the message property of the DOMException is not writable, which leads to this error:

```
layer.ts:549 Uncaught (in promise) TypeError: Cannot set property message of  which has only a getter
    at BitmapLayer.raiseError (layer.ts:549:1)
    at LayerState._onError (layer-state.ts:93:1)
    at component-state.ts:265:1
```

https://github.com/visgl/deck.gl/blob/818751a9af76b41cfdea020a9dff06bf409c6fac/modules/core/src/lib/layer.ts#L549

Perhaps instead of trying to modify the original object, a new exception can be created, or use [VError](https://github.com/TritonDataCenter/node-verror) to chain the exception 

### Flavors

- [ ] Script tag
- [X] React
- [ ] Python/Jupyter notebook
- [ ] MapboxOverlay
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] ArcGIS

### Steps to Reproduce

Abort a resource fetch using an abort controller.

### Environment

- Framework version: 8.9.7
- Browser: Version 114.0.5735.90 (Official Build) (arm64)
- OS: MacOS

**Comment by Pessimistress** [comment]
Fixed in v8.9.20.

**Comment by ricardoekm** [comment]
Awesome, thanks @Pessimistress .
